### DISCUSSION TITLE
요금제 관련 문의 드립니다.

### DISCUSSION THREAD

**Discussion by billyLee1004** [commerce-api-naver/commerce-solutions#54] [묻고 답하기]
안녕하세요 에이아이닷엠 빌리 입니다. 

요금제 관련하여 문의드립니다 .

예를들어
요금제 베이직 26,900원 PlanId : TEST111  상품이 있는데 마곡수산이라는 쇼핑몰운영자분이 월 구독을 하셨어요 
구독중에 베이직 요금이 잘 못되어 새로운 베이직 요금 30,000원 PlanId : TEST222 으로 새로운 요금제를 추가할 경우 기존 요금(TEST111)은 비노출(노출x) 
마곡수산이라는 쇼핑몰 운영자는 다음 결제일에 어떤 요금제로 결제하게 되는지 문의 드립니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

구독자가 구독중인 요금제는 변동되지 않습니다.
요금제명이 동일한 '베이직'이더라도 요금제ID(`planId`)가 다르다면 시스템적으로는 서로 다른 요금제로 취급합니다.
따라서 예시로 들어주신 '마곡수산' 구독자는 노출되지 않는 이전 상품(plainId: TEST111) 조건으로 계속 정기 결제가 진행됩니다.
